src/aap_eda/services/activation/activation_manager.py

```diff
@@ -207,6 +207,13 @@ def _start_activation_instance(self):
 
         self.db_instance.refresh_from_db()
         log_handler = self.container_logger_class(self.latest_instance.id)
+        if self.db_instance.edited_at:
+            edited_at = self.db_instance.edited_at.strftime(
+                "%Y-%m-%dT%H:%M:%S.%fZ"
+            )
+            log_handler.write(
+                f"The activation was edited at {edited_at}", flush=True
```

Comment:
@bzwei @hsong-rh How would this be handled in the UI, should the UI display a separate field which is TZ aware when displaying the Edited At. The logs are good for record keeping, so we should keep this.

Reply:
@bzwei Can you paste a screen shot of the Activation log in this PR?

Reply:
@bzwei To be consistent can be use the created_at if the edited_at is missing that way the log will remain the same.

Reply:
@mkanoor The edited_at timestamp is not the message logging time. It is part of the log message, just like other timestamps appear in the output. Screenshot is posted.
I purposely not log the edited time if not present. When it is missing, it is an indication that the activation is original, never updated. Or perhaps I can write a message here "The activation has never been edited"

Reply:
@bzwei Thats ok leave it the way it is for now. Did you forget to the add the screen shot?